syntax = "proto3";

import "google/protobuf/empty.proto";
import "google/protobuf/duration.proto";

message RobotProxyPos {
  int32 first = 1;
  int32 second = 2; 
}

// The greeting service definition.
service RobotProxy {
  rpc tell (google.protobuf.Empty) returns (RobotProxyPos);
  rpc moveTo (RobotProxyPos) returns (google.protobuf.Empty);
  rpc fire (google.protobuf.Duration) returns (google.protobuf.Empty);
}
